synapse/http/servlet.py

```diff
@@ -131,7 +131,6 @@ def parse_string(
             string, or None if any value is allowed, defaults to None. Must be
             the same type as name, if given.
         encoding (str|None): The encoding to decode the string content with.
```

Comment:
Can you remove the types from the docstring please? (It stops them from getting out of sync)